src/client/get.ts

```diff
@@ -47,7 +47,9 @@ export type GetAptosRequestOptions = Omit<GetRequestOptions, "type">;
  * @param options GetRequestOptions
  * @returns
  */
-export async function get<Req, Res>(options: GetRequestOptions): Promise<AptosResponse<Req, Res>> {
+export async function get<Req extends {}, Res extends {}>(
```

Comment:
why do you add extends everywhere..?

Reply:
The generics have to be objects, rather than just any input. There's stronger typing in future versions of some of these